### PR TITLE
dao8 done

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,6 +29,12 @@
             <version>4.0.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>jstl</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
+        </dependency>
+
```
Comment (Max-Pochepets) [inline pom.xml]: redundant empty line

File: src/main/webapp/WEB-INF/views/cars/create.jsp [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+<h1>Please provide information about car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/create">
+    Enter car model: <input type="text" name="model">
+    Enter manufacturer ID: <input type="number" name="manufacturer">
+    <button type="submit">Create</button>
+</form>
+</body>
+</html>
```
Comment (Max-Pochepets) [inline src/main/webapp/WEB-INF/views/cars/create.jsp]: make sure you have empty lines in the end of all your files

File: src/main/webapp/WEB-INF/views/cars/drivers/add.jsp [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Please provide information about driver and car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Enter car id: <input type="number" name="car_id">
```
Comment (Max-Pochepets) [inline src/main/webapp/WEB-INF/views/cars/drivers/add.jsp:9]: what if I input nothing in this field?

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/core/controller/AddDriverToCarController.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            Long driverId = Long.valueOf(req.getParameter("driver_id"));
+            Car car = carService.get(carId);
+            Driver driver = driverService.get(driverId);
+            carService.addDriverToCar(driver, car);
+            resp.sendRedirect(req.getContextPath() + "/");
+        } catch (NoSuchElementException e) {
+            req.setAttribute("message", "Incorrect data input. "
+                    + "Check if driver/car with provided id exists");
+            req.getRequestDispatcher("/WEB-INF/views/cars/drivers/add.jsp").forward(req, resp);
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controller/AddDriverToCarController.java]: let's not catch runtime exceptions. In this task we can just "believe" that we will get a proper id

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/views/cars/drivers/add.jsp (hunk 3 above)
Comment (dmytro-hryhoruk) [reply src/main/webapp/WEB-INF/views/cars/drivers/add.jsp:9]: let's use required attribute for each input

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**